.github/workflows/publish.yaml

```diff
@@ -23,34 +23,24 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v2
         with:
-          node-version: '16'
+          node-version: "16"
 
-      - name: 'Install'
-        run: npm ci
+      - name: Update version
+        id: update-version
+        run: |
+          cd libs/react-numerics
+          git tag $(npm version ${{ github.event.inputs.update_type }} ) && git add package.json
+          cd -
 
-      - name: Create TypeDocs
-        run: npx typedoc
+      - name: "Install"
+        run: npm ci
 
-      - name: 'Build'
+      - name: "Build"
         run: npm run build
 
-      - name: 'Test'
+      - name: "Test"
         run: npm run test
 
-
-      - name: Increment Version
```

Comment:
We don't need the typedocs, at least not long term, but this increment-version step is the primary point of this workflow. We _do_ need this. If we don't have it, the next step will fail because the version number hasn't changed.